content/en/blog/_posts/2023-07-24-mixed-version-proxy-alpha.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Upgrade Kubernetes more safely by enabling Mixed Version Proxy (Alpha)"
+date: 2023-05-24
+slug: kubernetes-1-28-feature-mixed-version-proxy-alpha
+---
+
+**Author:** Richa Banker (Google)
+
+This blog describes “Mixed Version Proxy”, a new alpha feature in Kubernetes 1.28. Mixed Version Proxy enables a resource request to be served by the correct apiserver in cases where there are multiple apiservers at varied versions in a cluster, ex: during an upgrade of a cluster, when a runtime config changes and a rollout occurs.
+
+## What problem does this solve?
+When a cluster undergoes an upgrade, the kube-apiservers existing at different versions in that scenario can serve different sets (groups, versions, resources) of built-in resources. A resource request made in this scenario may be served by any of the available apiservers, potentially resulting in the request ending up at an apiserver that may not be aware of the requested resource; consequently it being served a 404 not found error which is incorrect. Furthermore, incorrect serving of the 404 errors can lead to serious consequences such as namespace deletion being blocked incorrectly or objects being garbage collected mistakenly.
+
+## How do we solve the problem?
+
+![Alt text](https://github.com/kubernetes/website/assets/26771552/5c0fbb01-5e30-467d-81bf-5739be19b013?raw=true "MVP-flow-diagram")
```

Comment:
- We recommend a figure shortcode.
- Please link to the rendered SVG asset so that Netlify CDN can serve it

Reply:
Added a figure shortcode.